samples/snippets/csharp/VS_Snippets_CFX/CFX_CompensationExample/cs/Program.cs

```diff
@@ -261,9 +261,7 @@ static void RunWF(Activity wf)
             {
                 if (e.TerminationException != null)
                 {
-                    Console.WriteLine("Workflow terminated with exception:\n{0}: {1}",
-                        e.TerminationException.GetType().FullName,
-                        e.TerminationException.Message);
+                    Console.WriteLine($"Workflow terminated with exception:\n{e.TerminationException.GetType().FullName}: {e.TerminationException.Message}");
```

Comment:
I'd use raw string literals here:

```suggestion
                    Console.WriteLine($"""
                    Workflow terminated with exception:
                    {e.TerminationException.GetType().FullName}: {e.TerminationException.Message}
                    """);
```